pyhanko/sign/fields.py

```diff
@@ -678,6 +678,17 @@ class SigSeedSubFilter(Enum):
     ETSI_RFC3161 = pdf_name("/ETSI.RFC3161")
 
 
+@unique
+class SigSeedAuthType(Enum):
```

Comment:
Best to remove the `Seed` here, since IIRC this property is not one of the ones you can set in the seed value dictionary.

(Granted, the naming on `SigSeedSubFilter` is kinda bad too, since using it in a seed value dictionary is far from the most common use case... I'll have to do something about that.)

Reply:
It's done.